samples/app_event_manager/sample.yaml

```diff
@@ -58,13 +58,15 @@ tests:
       - nrf54h20dk/nrf54h20/cpuapp
     extra_configs:
       - CONFIG_SHELL=y
```

Comment:
Commit message title: `nativ_sim` -> `native_sim`

Reply:
done. Thank you for vigilance :)